package.json

```diff
@@ -24,6 +24,7 @@
     "bcrypt": "^5.0.0",
     "bootstrap": "^5.1.3",
     "compression": "^1.7.3",
+    "dotenv": "^16.0.0",
```

Comment:
Hey @ssalim5 , is this for storing passwords?

If so, will the code break if the rest of us don't have the passwords?